src/platform/graphics/constants.js

```diff
@@ -419,42 +419,42 @@ export const PIXELFORMAT_L8 = 1;
  *
  * @type {number}
  */
-export const PIXELFORMAT_L8_A8 = 2;
+export const PIXELFORMAT_LA8 = 2;
```

Comment:
I think these constants should be properly deprecated? Either keep them here (in addition to new ones), or add them to deprecated, but not to gfx name space as currently.

Reply:
actually, I take it back, there are in deprecated in both places. All good.